fission-core/library/Fission/Internal/Orphanage/RSA2048/Public.hs

```diff
@@ -32,9 +34,8 @@ instance Arbitrary RSA.PublicKey where
 
 instance Display RSA.PublicKey where
   textDisplay pk =
-    X509.PubKeyRSA pk
-      |> X509.pubKeyToPEM
-      |> PEM.pemWriteBS
+    [X509.PubKeyRSA pk]
+      |> X509.writePubKeyFileToMemory
```

Comment:
These end up _larger_ than before.

> `writePubKeyFile path = writePEMs path . map pubKeyToPEM`

Soooo, that's in PEM, not X.509. This is definitely wrong.